src/components/dev-hub/card-badge.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { colorMap, size } from './theme';
+import { P5 } from './text';
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+
+const CardBadge = styled('div')`
+    align: left;
+    bottom: 0;
+    background-color: ${colorMap.greyDarkThree};
+    display: inline-block;
+    font-family: 'Akzidenz-Grotesk';
+    margin: ${size.default};
+    padding: 1px ${size.xsmall};
+    position: absolute;
+    text-transform: uppercase;
+    ${props =>
+        props.color &&
+        css`
+            border: 1px solid ${props.color};'
+        `}
```

Comment:
We actually don't need `css` here and can use destructuring on props to get right to `color`, this can be `$({(color}) => color && 'border: 1px solid ${color}';` (with backticks, the formatting is getting messed up on my comment)